ci/install.sh

```diff
@@ -117,11 +117,11 @@ if [ ! -e "${GPG21_INSTALL}/bin/gpg" ]; then
   ${MAKE} -j${CORES} install
   cd ..
 
-  wget -c https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.1.23.tar.bz2
-  wget -c https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.1.23.tar.bz2.sig
-  gpg --verify gnupg-2.1.23.tar.bz2.sig
-  tar -xjf gnupg-2.1.23.tar.bz2
-  cd gnupg-2.1.23
+  wget -c https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.4.tar.bz2
```

Comment:
Or we could fetch the latest one?

Reply:
Unfortunately they don't have link  to "latest"

404 Not Found
The requested URL /ftp/gcrypt/gnupg/gnupg-latest.tar.bz2 was not found on this server.

Reply:
Maybe we need a script to parse the [site](https://www.gnupg.org/ftp/gcrypt/gnupg/) and extract the highest version number... maybe @erikbor has an easy solution?

Reply:
I had this thought - I would rather parse this https://gnupg.org/download/index.html.
But then we depend on format of the site and build may break if format changes (small chances). Why not...

Reply:
I think @flowher's idea seems like the simplest if they don't have any nice/official way of doing this. Some python w/html5lib and some xpath or something could work.

We could also use the most recent tags from the git repos or something like that, but the download page feels like a safer bet to me.